lib/elements/custom-style.js

```diff
@@ -75,7 +75,7 @@ export class CustomStyle extends HTMLElement {
    * @export
    * @return {HTMLStyleElement} This element's light-DOM `<style>`
    */
-  'getStyle'() {
+  ['getStyle']() {
```

Comment:
But why do we need this at all? What's wrong wrong with `getStyle()` and the `@export` we already have?

Reply:
Good point, export should be sufficient.  Will revert back to `getStyle()`

Reply:
`@export` requires both closure-library and the `--generate_exports` flag to be used with the compiler making it a not-great solution.

There are two options:

 1. Remove the `@export` annotation and quote the property directly.
 2. Manually export the method with a `CustomStyle.prototype['getStyle'] = CustomStyle.prototype.getStyle` statement right below the class definition.

Reply:
All of our internal users have `--generate_exports` enabled. We're also advocating for it to become enabled by default. So I think that's our preferred solution. Any reason you don't want to turn on the flag?

Reply:
Yes it requires closure library. That's a non starter for most external projects.

Reply:
Hm, seems odd. I thought it was just a JS compiler pass. Maybe it's like JSCompiler_renameProperty and just requires any implementation of the functions that it generates, since the compiler replaces the call anyway?

Reply:
It's very odd. The compiler pass literally adds `goog.exportSymbol` calls but requires the definition of that method to be included as source. Unlike property reflection, the source actually is present in the final output. 

The definition for that method could be injected like a polyfill, but nobody has done that work.